.env.example

```diff
@@ -0,0 +1,3 @@
+export GUARDIAN_SECRET_KEY=
```

Comment:
People (me) tend to assume this uses some sort of dotenv approach, where a 3rd party package automatically uses this file as the environment. Might not be a bad idea to add a comment here, explaining the user needs to `source` it, or add a clause to the readme explaining the same thing. Readme probably makes more sense.